coolq/bot.go

```
@@ -157,7 +157,12 @@ func (bot *CQBot) UploadLocalVideo(target int64, v *LocalVideoElement) (*message
 		if err != nil {
 			return nil, err
 		}
-		defer video.Close()
+		defer func() {
```

这类Close的err可以直接忽略 如 `defer func() {_ = video.Close()}()`

---

### EXC0001 errcheck: Almost all programs ignore errors on these functions and in most cases it's ok
- Error return value of .((os\.)?std(out|err)\..*|.*Close|.*Flush|os\.Remove(All)?|.*print(f|ln)?|os\.(Un)?Setenv). is not check

可以把这条规则加入ignore中